src/univalue/include/univalue.h

```diff
@@ -302,6 +305,6 @@ static inline bool json_isspace(int ch)
 
 extern const UniValue NullUniValue;
 
-const UniValue& find_value( const UniValue& obj, const std::string& name);
+const UniValue& find_value(const UniValue& obj, const std::string& name, enum UniValue::keystatus *status=NULL);
```

Comment:
Are you sure this API change is really necessary? You could find out if the key is there with an additional call to `Univalue::Exists`? (or am I missing something)